search-index-import-util/src/test/resources/datapipeline_output_sample

```diff
@@ -0,0 +1,46 @@
+{"owner":{"s":"sase@unit.no"},"identifier":{"s":"9b98c263-6974-49bc-b42b-2fa9562c6432"},"publisherId":{"s":"https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934"},"createdDate":{"s":"2020-09-28T11:57:31.466056Z"},"publisherOwnerDate":{"s":"https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934#sase@unit.no#2020-09-28T11:57:31.466056Z"},"modifiedDate":{"s":"2020-09-28T11:57:31.466056Z"},"publisher":{"m":{"id":{"s":"https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934"},"type":{"s":"Organization"}}},"type":{"s":"Publication"},"status":{"s":"Draft"},"fileSet":{"m":{"files":{"l":[{"m":{"identifier":{"s":"e0cf7d7a-f636-4bc0-ac9c-fcbdddfd4c8f"},"size":{"n":"164161"},"publisherAuthority":{"bOOL":false},"name":{"s":"Skjermbilde 2020-09-25 kl. 11.28.22.png"},"administrativeAgreement":{"bOOL":false},"mimeType":{"s":"image/png"},"type":{"s":"File"}}}]},"type":{"s":"FileSet"}}}}
```

Comment:
Should this file have extension ".json"?

Reply:
Maybe - It's not valid json, it is just json stacked in a file  (thanks to DataPipeline)